### PR TITLE
Fixed: CSS-Animation w/ iteration-count:infinite is never reset causing out of memory

### PR DESCRIPTION
**Pull request by tzraikov**
#2209

### CODE REVIEW

File: tns-core-modules/ui/core/view.android.ts [hunk 1]
```diff
@@ -127,6 +127,7 @@ export class View extends viewCommon.View {
 
     public onUnloaded() {
         super.onUnloaded();
+        this._unregisterAllAnimations();
```
Comment (hshristov) [inline tns-core-modules/ui/core/view.android.ts]: Can we make animations to listen for `Unloaded` and unregister?

Comment (tzraikov) [inline tns-core-modules/ui/core/view.android.ts]: Created another issue for that:
https://github.com/NativeScript/NativeScript/issues/2316

**Comment by vchimev** [comment]
run ci

**Comment by vchimev** [comment]
run ci

**Comment by dtopuzov** [comment]
run ci

**Comment by tzraikov** [comment]
run ci

**Pull request merged**